packages/apollo-server-core/src/plugin/schemaReporting/index.ts

```diff
@@ -2,6 +2,7 @@ import os from 'os';
 import type { InternalApolloServerPlugin } from '../internalPlugin';
 import { v4 as uuidv4 } from 'uuid';
 import { printSchema, validateSchema, buildSchema } from 'graphql';
+import { fetch } from 'apollo-server-env';
```

Comment:
this could be `import type`? (We didn't use `import type` in the past because we wanted to support older TS but have started using it recently.)